tests/operator_test.go

```diff
@@ -526,9 +563,19 @@ spec:
 			Expect(err).ToNot(HaveOccurred())
 		}
 
+		// repost original CDI object if it doesn't still exist
+		// in order to restore original environment
+		if originalCDI != nil {
+			_, err = virtClient.CdiClient().CdiV1alpha1().CDIs().Get(originalCDI.Name, metav1.GetOptions{})
+			if err != nil && errors.IsNotFound(err) {
+				createCdi()
+			} else {
+				Expect(err).ToNot(HaveOccurred())
```

Comment:
The CRD may be in DELETING state (delete timestamp != nil). If that's the case, should we try to wait for it to vanish, then createCdi()?

Reply:
This would likely only occur if someone attempted to cancel the testsuite.  I can add a check here to attempt to wait a bit for the deletion first though.